sidebars.js

```diff
@@ -35,6 +35,16 @@ module.exports = {
         },
       ],
     },
+    {
+      type: "category",
+      label: "Storage",
+      items: [
+        {
+          type: "autogenerated",
```

Comment:
the model cache page is at the top under `Storage`, maybe we should move it below the `create a network volume` section? Feel new users will want to look at the basic, we can add a (new) tag if you want this stands out.